mne/artifact_detection.py

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Nov 19 09:40:37 2019
+
+@author: an512
+"""
+
+from scipy.stats import zscore
+import mne
+import numpy as np
+
+from mne.annotations import Annotations
+from scipy.ndimage.measurements import label
+from mne.io.ctf.trans import _quaternion_align
+from mne.chpi import _apply_quat
+from itertools import compress
+
+
+def detect_bad_channels(raw, zscore_v=4, method='both', t1=30, t2=220,
```

Comment:
give `t1` and `t2` better names like `start` and `stop` or `duration`. Look around in the rest of the repository and don't be creative with your naming convention :)